tests/test_workers/test_pkg_managers/test_general_js.py

```diff
@@ -116,40 +111,49 @@ def test_download_dependencies(
             "cachito",
             custom_ca_path=None,
         )
-    mock_run_cmd.assert_called_once()
-    # This ensures that the bundled dependency is skipped
-    expected_npm_pack = [
-        "npm",
-        "pack",
-        "@angular-devkit/architect@0.803.26",
-        "@angular/animations@8.2.14",
-        "rxjs@6.5.5-external-gitcommit-78032157f5c1655436829017bbda787565b48c30",
-        "exsp@2.10.2-external-sha512-abcdefg",
-    ]
-    assert mock_run_cmd.call_args[0][0] == expected_npm_pack
-    run_cmd_env_vars = mock_run_cmd.call_args[0][1]["env"]
-    assert run_cmd_env_vars["NPM_CONFIG_CACHE"] == str(mock_td_path.join("cache"))
-    assert run_cmd_env_vars["NPM_CONFIG_USERCONFIG"] == mock_npm_rc_path
-    assert mock_run_cmd.call_args[0][1]["cwd"] == download_dir
 
-    dep1_source_path = os.path.join(download_dir, "angular-devkit-architect-0.803.26.tgz")
+    # dep_1
+    dep1_source_path = os.path.join(
+        download_dir,
+        "angular-devkit-architect-0.803.26.tgz",
+    )
     dep1_dest_path = os.path.join(
-        download_dir, "@angular-devkit/architect/angular-devkit-architect-0.803.26.tgz"
+        download_dir,
+        "@angular-devkit/architect/angular-devkit-architect-0.803.26.tgz",
+    )
```

Comment:
Did something change in the source/dest paths? If this is just a formatting change, could it be done in a separate commit?